fix 'cannot scream while unconscious' bug

checkraisefold opened the pull request:
:cl: zennerx
fix: fixed a bug that made you try and scream while unconscious due to a fire
/:cl:

so player's don't get spam saying you can't cough and scream while unconscious when on fire

checkraisefold committed:
Merge pull request #1 from tgstation/master

a

checkraisefold committed:
fix 'cannot scream while unconscious' bug 

fixes a bug where it tells you that you can't scream while unconscious while on fire (experienced on Bagil when a plasma fire happened at endround shuttle)

XDTM commented on code/modules/mob/living/carbon/human/species.dm:
```diff
@@ -1472,7 +1472,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		burn_damage *= heatmod
 		if((prob(burn_damage) * 10) / 4)	//40% for level 3 damage on humans
-			H.emote("scream")
+			if (h.stat != UNCONSCIOUS)
```
H instead of h

checkraisefold replied:
whoops.

checkraisefold committed:
Update species.dm

optimumtact commented:
For your changelog, the "optional name here" field can either be removed, in which case it will use your github username, or replaced with your chosen string.

you should not use "optional name here"

checkraisefold commented:
fixed @optimumtact

optimumtact commented:
Thanks and welcome to the repo

checkraisefold committed:
Merge pull request #2 from tgstation/master

update to current tgcode, avoid merge conflicts

ShizCalev commented on code/modules/mob/living/carbon/human/species.dm:
```diff
@@ -1472,7 +1472,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		burn_damage *= heatmod
 		if((prob(burn_damage) * 10) / 4)	//40% for level 3 damage on humans
-			H.emote("scream")
+			if (H.stat != UNCONSCIOUS)
```
Merge this into the above if

checkraisefold replied:
thought I did but missed it, fixing.

checkraisefold replied:
it's already merged into the above if, it's indented

vuonojenmustaturska replied:
he means change the if above to read `if(H.stat != UNCONSCIOUS && (prob(burn_damage) * 10) / 4))`, also you should change the condition to be either `!H.stat` or `H.stat > UNCONSCIOUS`, the former is a fairly generic "is not dead, unconsciout or in crit" check, the latter allows for soft crit as well as conscious.

checkraisefold replied:
thanks! fixing it now. i'm pretty new to BYOND coding and especially to checking human properties. not familiar with stat, but familiar with if operators.

checkraisefold committed:
Merge pull request #3 from tgstation/master

update tg code

checkraisefold committed:
sanitize code

checkraisefold committed:
add the comment back so people know what it doin

checkraisefold committed:
change operator

other one wouldn't work with the fix, the screaming is supposed to happen while you're in crit
